src/pluralRules/pluralRule00.js

```diff
@@ -0,0 +1,6 @@
+// Families: Asian (Chinese, Japanese, Korean), Persian, Turkic/Altaic (Turkish), Thai, Lao
```

Comment:
For all these rules there isn't a sophisticated lib available ?

Reply:
I haven't checked actually. The main idea was to have small functions, so it could be easier to inline them and have a smaller bundle size as a result.